src/js/views/patients/schedule/schedule_views.js

```diff
@@ -149,34 +149,45 @@ const DayItemView = View.extend({
     'click .js-patient-sidebar-button': 'click:patientSidebarButton',
     'click .js-patient': 'click:patient',
     'click': 'click',
-    'click .js-select': 'click:select',
+  },
+  modelEvents: {
+    'change': 'render',
   },
   initialize({ state }) {
     this.state = state;
     this.flow = this.model.getFlow();
     this.isReduced = state.get('isReduced');
 
     this.listenTo(state, {
-      'select:all': this.render,
-      'select:none': this.render,
+      'select:all': this.showCheck,
+      'select:none': this.showCheck,
     });
   },
   onRender() {
-    this.toggleSelected(!this.isReduced && this.state.isSelected(this.model));
+    if (!this.isReduced) this.showCheck();
```

Comment:
Without `if (!this.isReduced)`, the reduced version of the schedule doesn't load any actions (without showing any errors). So something breaks for the reduced schedule if `showCheck()` is called.

Reply:
I think it might be better to throw a guard in the `showCheck`
Then you also wouldn't need the check to set `isSelected`

Reply:
Yes, good call 👍